pom.xml

```diff
@@ -550,9 +550,9 @@
             <id>rhproduct</id>
             <properties>
                 <quarkus.upstream.version>${quarkus.version}</quarkus.upstream.version>
-                <quarkus-bom.version>${quarkus.upstream.version}-redhat-00001</quarkus-bom.version>
+                <quarkus-bom.version>${quarkus.upstream.version}</quarkus-bom.version>
                 <platform.groupId>com.redhat.quarkus.platform</platform.groupId>
-                <platform.version>${project.version}-redhat-00001</platform.version>
+                <platform.version>${project.version}</platform.version>
```

Comment:
@harshmadhani these two properties can be removed completely from this profile.
`platform.version` is already initialized to the `project.version` and the `quarkus-bom.version` should be set to a redhat version when you build it.

Reply:
But is not the quarkus-bom.version used as reference at some places in the pom?

Reply:
It is. You should not remove it completely from the pom but from the rhproduct profile.

Reply:
Only the property overrides.